docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -307,6 +339,24 @@ The Observability AI Assistant connector is called when the alert fires and when
 
 To learn more about alerting, actions, and connectors, refer to <<create-alerts>>.
 
+[discrete]
+[[obs-ai-settings]]
+== AI Assistant Settings
+
+You can access the AI Assistant Settings page:
```

Comment:
Perhaps just personal preference, but in documentation, the phrase "you can" is almost always unnecessary.

_You can_ be more direct by saying:

`Access the AI Assistant Settings page:`